Yoast/YoastSEO.js
# Add functionality to recognize German adjective forms

## Summary

This PR can be summarized in the following changelog entry:

* [not-user-facing] Adds functionality to create German adjective forms.

## Relevant technical choices:

* For nouns, we check exceptions by matching the exception stems at the end of the stemmed word. This is because nouns are commonly are used in compounds (e.g. `stadt` in `Hauptstadt`). Since this is far less common for adjectives, I decided to match exceptions with the full stemmed word.
* I added some function words we came across while researching adjectives.

## Test instructions

This PR can be tested by following these steps:

* Make sure you have the premium configuration set to the right branch (see https://github.com/Yoast/YoastSEO.js-premium-configuration/pull/7).
* Change the language of your site to German.
* Add a German text and a keyphrase including German adjectives.
* Add different grammatical forms of the adjective in your text; make sure these forms are recognized.

Fixes #2127
